java-algs/src/test/java/edu/algs/tree/BSTreeTest.java

```diff
@@ -198,7 +198,7 @@ private void ordered(int d, int relaxation, boolean debug) {
 
     private void random(int d, int relaxation, boolean debug) {
         int n = (1 << d) - 1;
-        BinTree<Integer> tree = BinTree.empty();
+        BSTree<Integer> tree = new BSTree<>();
```

Comment:
When using balanced tree, balancing tests show much lower tree growth (e.g. balanced tree is being balanced better than unbalanced, duh).